resources/META-INF/plugin.xml

```diff
@@ -39,15 +40,17 @@
   <depends>com.intellij.modules.lang</depends>
 
   <actions>
-    <action id="ClangFormat.ClangFormat" class="io.probst.idea.clangformat.ClangFormatAction"
-            text="clang-format"
-            description="Runs clang-format on this buffer">
-      <add-to-group group-id="EditorPopupMenu" anchor="last"/>
-
-      <keyboard-shortcut keymap="Mac OS X" first-keystroke="control alt K"/>
-      <keyboard-shortcut keymap="Mac OS X" first-keystroke="meta alt K"/>
-      <keyboard-shortcut keymap="Mac OS X 10.5+" first-keystroke="meta alt K"/>
-    </action>
+    <group id="ClangFormat.Group" text="ClangFormat" popup="true">
+      <action id="clangFormat" class="io.probst.idea.clangformat.ClangFormatAction" text="_Format this file"
+              description="Format this file">
+        <keyboard-shortcut keymap="Mac OS X" first-keystroke="control alt K"/>
+        <keyboard-shortcut keymap="Mac OS X" first-keystroke="meta alt K"/>
+        <keyboard-shortcut keymap="Mac OS X 10.5+" first-keystroke="meta alt K"/>
+
+      </action>
+      <add-to-group group-id="CodeMenu"/>
```

Comment:
What's the difference between CodeMenu and EditorMenu? Should we add this to both?